### PR TITLE
assigning privilege now takes user global Id

### PR DESCRIPTION
**Pull request by mannSheth**
Needs to be deployed at the same time as https://github.com/AusTrakka/austrakka-portal/pull/1900

### CODE REVIEW

File: austrakka/utils/subcommands/privilege.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ def privilege_list_by_user(user_id: str, record_global_id: str, out_format: str)
     @opt_role()
     @opt_record_global_id()
     def privilege_assign(
-            user_id: str,
+            user_global_id: str,
             role: str,
             record_global_id: str):
         record_type_route = convert_record_type_to_route_string(roottype)
-        assign_privilege(user_id, role, record_global_id, record_type_route)
+        assign_privilege(user_global_id, role, record_type_route, record_global_id)
```
[the remark this answers is not in the document]
Comment (mannSheth) [reply austrakka/utils/subcommands/privilege.py:77]: Thats odd, This is my mistake but I don't understand how the privilege assignment works. When manually testing the privilege assignment, it works fine.

File: austrakka/components/iam/privilege/__init__.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
     opt_record_type,
     opt_role,
     opt_user_object_id,
-    opt_record_global_id)
+    opt_record_global_id, opt_user_global_id)
```
[the remark this answers is not in the document]
Comment (mannSheth) [reply austrakka/components/iam/privilege/__init__.py]: Done

File: austrakka/utils/subcommands/privilege.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mannSheth) [reply austrakka/utils/subcommands/privilege.py]: Fixed

**Comment by claresloggett** [comment]
> BTW, why is this PR blocked?

Because it had a companion PR that wasn't ready

**Pull request merged**